pkg/storkctl/actionstatus_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build unittest
+// +build unittest
+
+package storkctl
+
+import (
+	"fmt"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetFailoverNoFailoverAction(t *testing.T) {
+	cmdArgs := []string{"get", "failover"}
+	expected := "No resources found.\n"
+	var actionList storkv1.ActionList
+	testCommon(t, cmdArgs, &actionList, expected, false)
+}
+
+func TestGetFailoverOneSuccessfulFailoverAction(t *testing.T) {
+	defer resetTest()
+	// create a failover action
+	actionName := createFailoverActionAndVerify(t)
+	// update the status pf the action
+	actionObj, err := storkops.Instance().GetAction(actionName, "kube-system")
+	require.NoError(t, err, "Error getting action")
+	actionObj.Status.Stage = storkv1.ActionStageFinal
+	actionObj.Status.Status = storkv1.ActionStatusSuccessful
+	actionObj.Status.Summary = &storkv1.ActionSummary{FailoverSummaryItem: []*storkv1.FailoverSummary{
+		{
+			Namespace: "ns1",
+			Status:    storkv1.ActionStatusSuccessful,
+			Reason:    "scaling up apps in namespace ns1 successful",
+		},
+		{
+			Namespace: "ns2",
+			Status:    storkv1.ActionStatusSuccessful,
+			Reason:    "scaling up apps in namespace ns2 successful",
+		},
+	}}
+	actionObj.Status.Reason = ""
+	_, err = storkops.Instance().UpdateAction(actionObj)
+	require.NoError(t, err, "Error updating action")
+	cmdArgs := []string{"get", "failover", actionName, "-n", "kube-system"}
+	expected := "NAME                                                CREATED   STAGE       STATUS       ADDITIONAL INFO\n" +
+		"failover-test-migrationschedule-2024-01-01-000000             Completed   Successful   Scaled up Apps in : 2/2 namespaces\n"
+	testCommon(t, cmdArgs, nil, expected, false)
+}
+
+func TestGetFailoverOneFailedFailoverAction(t *testing.T) {
```

Comment:
Add following tests
1. where out of 2 namespaces, one got successfully activated and one failed.
2. Create a test with verifying get calls outputs for various intermittent stages.

Reply:
makes sense. will address